canonical-websites/snapcraft.io
# Make snapracft as a module

# Summary

Make s.io as a module: move all files to `webapp` folder
Fixes #554 

# QA

- `./run`
- http://0.0.0.0:8004
- Make sure everything still works
- Create docker image: 
- `docker build --build-arg COMMIT_ID=test --tag test-sio .`
- Make sure it still works
